src/BlockTypes/ProductGallery.php

```diff
@@ -30,16 +23,26 @@ protected function get_block_type_style() {
 	 * @return string Rendered block type output.
 	 */
 	protected function render( $attributes, $content, $block ) {
+		$post_id                = $block->context['postId'] ?? '';
+		$product_gallery_images = ProductGalleryUtils::get_product_gallery_images( $post_id, 'thumbnail', array() );
+		$classname_single_image = '';
+
+		if ( count( $product_gallery_images ) < 2 ) {
+			// The gallery consists of a single image.
+			$classname_single_image = 'is-single-product-gallery-image';
+		}
+
 		$classname          = $attributes['className'] ?? '';
-		$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => trim( sprintf( 'woocommerce %1$s', $classname ) ) ) );
+		$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => trim( sprintf( 'wc-block-product-gallery %1$s %2$s', $classname, $classname_single_image ) ) ) );
```

Comment:
![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/a1c1747b-5821-44f3-90b0-886626c97410)

`wp-block-woocommerce-product-gallery` and `wc-block-product-gallery` are duplicated. I think that we should remove them on this div.